bin/verifyWeeSchema.js

```diff
@@ -34,16 +43,42 @@ for (const i in proposed) {
     }
 
     // checking test cases if any
-    if (!type.testcase) continue;
-    valid = ajv.validate(type, type.testcase);
-    if (!valid) {
-      clean = false;
-      console.error(
-        `# testcase failed IN ${i}:${j} ${ns.namespace}.${type.name} "${
-          type.id
-        }"`,
-      );
-      console.error(ajv.errors);
+    const testcases = type.testcases || [];
+    if (type.testcase) testcases.push(type.testcase);
+
+    if (!testcases.length) continue;
+
+    for (const tc of testcases) {
+      valid = ajv.validate(type, tc);
+      if (!valid) {
+        clean = false;
+        console.error(
+          `# testcase failed IN ${i}:${j} ${ns.namespace}.${type.name}  "${
+            type.id
+          }"
+
+${full(tc)}
+          `,
+        );
+        console.error(ajv.errors);
+      }
+    }
+
+    // now, known failures
+    for (const tc of type.failcases || []) {
```

Comment:
This is confusing to me... when would `const tc of type.failcases` be falsy? Could the default of that be set to `[]` instead?